core/controllers/platform_feature.py

```diff
@@ -29,16 +29,39 @@ class PlatformFeaturesEvaluationHandler(base.BaseHandler):
     """The handler for retrieving feature flag values."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
+            'platform_type': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
```

Comment:
`platform_type` should have choices.

Reply:
Based on [these comments in domain layer validate method](https://github.com/oppia/oppia/blob/37c24c46e6fc5a9ee1e8df6fb62d55737fa1ced3/core/domain/platform_parameter_domain.py#L123) they are not part of validation so if invaid  platform_type received then error should not be raised. 
If I only allow list of platform_types then [this test_case](https://github.com/oppia/oppia/blob/37c24c46e6fc5a9ee1e8df6fb62d55737fa1ced3/core/controllers/platform_feature_test.py#L98) should expected to raise 400 error instead of 200.
Please take a look.

Reply:
Hmm, okay.